Library/Formula/node-build.rb

```diff
@@ -0,0 +1,22 @@
+class NodeBuild < Formula
+  desc "Install NodeJS versions"
+  homepage "https://github.com/OiNutter/node-build"
+  url "https://github.com/OiNutter/node-build/archive/v2.0.1.tar.gz"
+  sha256 "21d62820fb918cb4866d315f9f0c157b655d994857949f179a58756c6dde7d14"
+  head "https://github.com/OiNutter/node-build.git"
+
+  bottle :unneeded
+
+  depends_on "autoconf" => [:recommended, :run]
+  depends_on "pkg-config" => [:recommended, :run]
+  depends_on "openssl" => :recommended
+
+  def install
+    ENV["PREFIX"] = prefix
+    system "./install.sh"
+  end
+
+  test do
+    system "#{bin}/node-build", "--version"
```

Comment:
Sure. I mostly just copied the formula from ruby build.

Are there any concerns around run time of the test suite? Will any
artifacts of the test run be cleaned up automatically or does that need
done in the test block?
On Nov 1, 2015 11:45 AM, "Mike McQuaid" <notifications@github.com> wrote:

> In Library/Formula/node-build.rb
> <https://github.com/Homebrew/homebrew/pull/45576#discussion_r43584505>:
>
> > +  sha256 "21d62820fb918cb4866d315f9f0c157b655d994857949f179a58756c6dde7d14"
> > +  head "https://github.com/OiNutter/node-build.git"
> > +
> > +  bottle :unneeded
> > +
> > +  depends_on "autoconf" => [:recommended, :run]
> > +  depends_on "pkg-config" => [:recommended, :run]
> > +  depends_on "openssl" => :recommended
> > +
> > +  def install
> > +    ENV["PREFIX"] = prefix
> > +    system "./install.sh"
> > +  end
> > +
> > +  test do
> > +    system "#{bin}/node-build", "--version"
>
> Could you maybe try and build a version of node here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/45576/files#r43584505>.
>